linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/ignition/shared.yml

```diff
@@ -1,10 +1,6 @@
 # platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ocp
```

Comment:
The only question about this whole PR is that the `no_empty_passwords` contains an enumeration of all the platforms..

Reply:
It can be ` # platform = multi_platform_all `. The `rule.yml` for `no_empty_passwords` doesn't define any `prodtype`.